### ISSUE TITLE
multiple hypothesis testing problem

### ISSUE THREAD

**Issue by austindenteh** [grf-labs/grf#1210]
Hi! Is inference on the heterogenous effects estimated using GRF susceptible to multiple hypothesis testing concerns? That is, does one need to perform any adjustments in this regard?

**Comment by erikcs** [comment]
That depends on what exactly what you are trying to do. You could for example estimate CATE on a training set and evaluate [RATE](https://grf-labs.github.io/grf/articles/rate.html) on a test set and to assess heterogeneity and be confident you're not attributing noisy overfit CATE estimates to false heterogeneity.
